CHANGELOG.md

```diff
@@ -11,6 +11,79 @@ All notable changes to Botonic will be documented in this file.
   </summary>
 </details>
 
+## [0.14.0] - 2020-30-09
+
+### Added
+
+- [@botonic/cli](https://www.npmjs.com/package/@botonic/cli)
+
+  - New bot template `dynamo` using Botonic dynamoDB plugin from a JS bot.
+
+* [@botonic/react](https://www.npmjs.com/package/@botonic/react)
+
+  - Improved behavior when end user messages are lost due to connection issues. Missed inputs will be resent when connection is regained.
+  - End user messages will be displayed with less opacity until they are correctly delivered to the server.
+  - Added classNames to blob messages by default, under the following format:
+    `{type}-{from} {customTypeName}`
+    e.g.: Text messages from user -> `text-user`
+    e.g.: Custom messages from bot -> `custom-bot my-custom-message-name`
+  - Passing `mobileStyle` in `webchat/index.js`is now allowed.
+  - Added arrow buttons for better interaction with Webchat Carousels. They can be customized.
+  - Disabling timestamps in the custom Messages using the `enableTimestamps` prop is now allowed.
+  - Defined where to store the botonicState using the variable storage.
+  - The methods `openCoverComponent`, `closeCoverComponent` and `toggleCoverComponent` are available in the browser, through the Botonic object, so that the `coverComponent` can be shown/hidden manually.
+
+- [@botonic/plugin-contentful](https://www.npmjs.com/package/@botonic/plugin-contentful)
+
+  - Decorator for Contentful that can distribute the delivery requests amongst different spaces/environments.
+  - Added a new tool to modify Contentful spaces exported as json file with "contentful space export".
+  - Added a tool to duplicate the value of reference fields into a new locale. Reference fields (assets or links to other contents) will typically be the same for all locales, but not always. So we initially link them all to the same target for all locales.
+  - Added support for locales which include the country (eg. es_ES). Now they are preprocessed as the root locale (without the country).
+  - Added an option for Contentful plugin to be able to transform the name of the locale when sent to Contentful.
+  - Added `logCall` flag to `contentfulOptions` in order to log creation config and calls performed to CMS.
+  - Added NLP support for Turkish.
+  - Added NLP support for Italian.
+  - Added Tokenizer, Lexer & Stop words for French.
+
```

Comment:
Added class ContentsValidator, which validates that all contents of a locale can be correctly delivered.